spec/reports/freq_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "reports/cost_report"
+require "clustering/cluster_holding"
+require "clustering/cluster_ht_item"
+
+RSpec.describe Reports::CostReport do
```

Comment:
The PR description says something about `FrequencyTable` but I don't yet see it here -- not yet committed?

Reply:
Do we want FrequencyTable in its own class for the purposes of this branch? My description is intended to point out the repurposable specs that were added.

Reply:
I don't think we need to do it on this branch, no.